src/Unit/Volume/CubicCentimeter.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types = 1);
+
+/**
+ * This file is part of the jordanbrauer/unit-converter PHP package.
+ *
+ * @copyright 2018 Jordan Brauer <18744334+jordanbrauer@users.noreply.github.com>
+ * @license MIT
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace UnitConverter\Unit\Volume;
+
+/**
+ * Cubic centimeter unit data class.
+ *
+ * @version 2.0.0
+ * @since 0.0.1
+ * @author Jordan Brauer <18744334+jordanbrauer@users.noreply.github.com>
```

Comment:
update the `@version` to `1.0.0` and the `@since` to `0.8.5`, and also feel free to put your own name/username and email here.